luigi/scheduler.py

```diff
@@ -656,8 +668,11 @@ def add_task(self, task_id=None, status=PENDING, runnable=True,
 
             # Task dependencies might not exist yet. Let's create dummy tasks for them for now.
             # Otherwise the task dependencies might end up being pruned if scheduling takes a long time
+            deps_retry_policy_dicts = _get_default(deps_retry_policy_dicts, {})
```

Comment:
Yes, it is global and luigi scheduler module has that method already and I just used it to get default value of a variable if it is `None`